utils/githooks/README.md

```diff
@@ -8,68 +8,88 @@ The DAOS repo contains several built-in githooks that are intended
 
-To enable these standard githooks requires a two step process:
+## Install DAOS Git Hooks
 
-1. Install the hooks
+Installing is a two-step process:
 
-Configure your core.hookspath as follows (Recommended):
+### 1. Install the hooks
 
+Recommended: Configure your `core.hookspath`.  
+Any new githooks added to the repository will automatically run,
+but possibly require additional software to produce the desired effect.
+Additionally, as the branch changes, the githooks change with it.
 ```sh
 git config core.hookspath utils/githooks
 ```
 
 Additionally, one can copy the files into an already configured path.
 
-With the first option, any new githooks added to the repository will
-automatically run, but possibly require additional software to produce the
-desired effect.  Additionally, as the branch changes, the githooks
-change with it.
-
-2. Install all of the required tools
+### 2. Install the tools
 
 The Githooks framework in DAOS is such that the hooks will all run.
 However, some hooks will simply check for required software and are
 effectively a noop if such is not installed.
 
-On many systems, the required packages can be installed through standard means
-but customization may be required.  Some are specified in
-[requirements.txt](../../requirements.txt) so can be installed using
-`pip install -r requirements.txt` and `pip install -r utils/cq/requirements.txt`
-which can also be done using a virtual environment. The following
-packages are used by built-in githooks.
+Requirements come from a combination of `pip` and system packages and can usually be installed through standard means.  
+To install `pip` packages specified in [utils/cq/requirements.txt](../../utils/cq/requirements.txt) it is recommended to setup a virtual environment and install with pip.  
+If you already have a [virtual environment for building](../../docs/QSG/build_from_scratch.md#python-packages) you can simply install the requirements:
+```sh
+python3 -m pip install -r utils/cq/requirements.txt
+```
+To install system packages with your package manager - for example:
+```sh
+sudo dnf install git-clang-format -y
+```
 
-1. clang-format version 14.0.5 or higher.  If the check is unable to parse
-the version output, it will fail.  Try running
-`<root>/site_scons/site_tools/extra/extra.py` to check.
-2. pylint
-3. flake8
-4. yamllint
-5. gofmt
+#### Installed tools
 
-There is a daos wrapper around pylint at `utils/cq/daos_pylint.py` which will perform standard
-pylint checks whilst managing scons and PYTHONPATH setup changes automatically.  Installing
-the python packages in `utils/cq/requirements.txt` will allow it to test for all the python dependencies.
+The following packages are used by built-in githooks.
 
-It is important to check the output on commit for any errors that may indicate
-any one of the required tools is missing.
+1. `clang-format` version 14.0.5 or higher - Formatter for C code. If the check is unable to parse
```

Comment:
```
1. `git-clang-format` and `clang-format` version 14.0.5 or higher - Formatter for C code.
If the check is unable to parse the version output, it will fail. Try running
`<root>/site_scons/site_tools/extra/extra.py` to check.
```
I think we should also mention `git-clang-format` here.